src/Dflydev/FigCookies/Cookie.php

```diff
@@ -42,46 +42,42 @@ public function withValue(?string $value = null) : Cookie
 
     /**
      * Render Cookie as a string.
-     *
      */
-    public function __toString() : string
+    public function __toString(): string
     {
         return urlencode($this->name) . '=' . urlencode((string) $this->value);
     }
 
     /**
      * Create a Cookie.
-     *
      */
-    public static function create(string $name, ?string $value = null) : Cookie
+    public static function create(string $name, ?string $value = null): Cookie
     {
-        return new static($name, $value);
+        return new self($name, $value);
```

Comment:
class is not final, means this is a behavior change

Reply:
Reverted.